src/plugins/data/public/ui/query_string_input/_query_bar.scss

```diff
@@ -40,6 +41,14 @@
     overflow-x: auto;
     overflow-y: auto;
     white-space: normal;
+    box-shadow: 0 0 0 1px $euiFormBorderColor;
+  }
+
+  @include euiFormControlWithIcon($isIconOptional: true);
+  ~ .euiFormControlLayoutIcons {
+    z-index: $euiZContentMenu + 1;
+    top: $euiSizeS + 3px;
```

Comment:
For pixel perfect calcs like these, I'd include a comment to explain why it exists.

Reply:
++ comment added